projects/angular-showcase/src/app/business/business-examples/status-showcase/status-showcase.component.html

```diff
@@ -1,3 +1,17 @@
+<h3>Status with type</h3>
+<p>
+  <sbb-status [type]="validType" [message]="'valid type'"></sbb-status>
```

Comment:
Here I would prefer assigning the type as an attribute, to show the simple usage
e.g. `<sbb-status type="valid" ...`